student-programs/champions/jahanvi-raycha.ftd

```diff
@@ -0,0 +1,81 @@
+-- import: fastn.com/featured as ft-ui
+-- import: fastn-community.github.io/spectrum-ds/common
+-- import: fastn-community.github.io/spectrum-ds as ds
+-- import: fastn-community.github.io/footer
+-- import: fastn-community.github.io/dark-flame-cs
+-- import: fifthtry.github.io/fastn-io-typography as typo
+
+
+
+
+-- ds.page: 
+show-footer: true
+site-logo: $fastn-assets.files.images.fastn.svg
+site-name: NULL
+logo-height: 38
+logo-width: 120
+colors: $dark-flame-cs.main
+types: $typo.types
+github-icon: true
+github-url: https://github.com/muskan1verma
+full-width: true
+
+-- ds.page.footer:
+
+-- footer.social-sideline-footer:
+social: true
+site-logo: $fastn-assets.files.images.fastn.svg
+site-url: /
+twitter-url: https://twitter.com/FifthTryHQ
+discord-url: https://discord.gg/bucrdvptYd
+copyright: Copyright © 2023 - [FifthTry.com](https://www.fifthtry.com/)
+
+-- end: ds.page.footer
+
+-- ds.contributor: Jahanvi Raycha
+avatar: $fastn-assets.files.images.students-program.champions.jahanvi-raycha.jpg
+profile: Champion
+connect: $social-links
+
+
+-- end: ds.page
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+-- common.social-media list social-links:
+
+-- common.social-media: 
+link: https://discord.com/invite/jahanvi
+src: $fastn-assets.files.images.discord.svg
+hover-src: $fastn-assets.files.images.discord-hover.svg
+
+-- common.social-media: 
+link: https://github.com/jahanvir
+src: $fastn-assets.files.images.github-grey.svg
+hover-src: $fastn-assets.files.images.github-grey-hover.svg
+
+-- common.social-media: 
+link: https://twitter.com/
```

Comment:
@jahanvir pl add your twitter profile id here.

Reply:
@gsalunke , I am not using twitter

Reply:
> @gsalunke , I am not using twitter

then, please, remove the link for now, in future if you start using it, you can raise a separate PR.